### PR TITLE
Feat #216 add project

### LINKED ISSUE #216: add more projects to /docs
Hi, we have a project-homepage here[1], and we need to add more projects to this page.

**Projects**
- [apisix-helm-chart](https://github.com/apache/apisix-helm-chart)
- [apisix-docker](https://github.com/apache/apisix-docker)

[1] https://apisix.apache.org/docs/

### PR DESCRIPTION
**Pull request by stu01509**
Fixes: #216

Changes:
Add the **helm-chart** project.

Screenshots of the change:

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -49,6 +49,17 @@ module.exports = {
...
+        name: "APISIX™ Helm Charts",
+        nameInParamCase: "helm-charts",
+        description: "An Apache APISIX Helm Charts provide the installation of Apache APISIX components for kubernetes.",
+        shape: "hexagon",
+        color: "#2563EB",
+        githubRepo: "apache/apisix-helm-chart",
+        version: "0.4.0",
+        releaseDate: "2021-03-12",
+        firstDocPath: "/apisix",
+      },
```
Comment (stu01509) [inline website/docusaurus.config.js:62]: Hi @juzhiyuan 

Could you help me to check this, I adjust the file path and structure, but I still not found the **helm-chart** page.

Comment (juzhiyuan) [reply website/docusaurus.config.js:62]: @stu01509 Hi, the docs will only appear in production env for now :)

**Review by juzhiyuan — APPROVED** [review]

File: website/docusaurus.config.js (hunk 1 above)
Comment (stu01509) [reply website/docusaurus.config.js:62]: OK, but could you confirm the shape, color, and releaseDate? because I didn't found the release date in the repo.

Comment (juzhiyuan) [reply website/docusaurus.config.js:62]: ok, this need @tokers's confirm

**Review by tokers — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**